icpc-wf/ansible/roles/scoreboard/tasks/main.yml

```diff
@@ -0,0 +1,119 @@
+---
+# These tasks configure a static scoreboard
+
+- name: create static scoreboard directory
+  file:
+    path: /home/domjudge/scoreboard
+    state: directory
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+
```

Comment:
```suggestion

```

If `directory', all intermediate subdirectories will be created if they do not exist.